internal/core/src/common/ChunkWriter.h

```diff
@@ -73,8 +76,8 @@ class ChunkWriter final : public ChunkWriterBase {
     ChunkWriter(int dim, bool nullable) : ChunkWriterBase(nullable), dim_(dim) {
     }
 
```

Comment:
Seems no place is using this offset, any ctx why we have this before. Maybe @sunby has some ideas